packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -468,61 +468,20 @@ export class SDKClient {
     const transactionType = transaction.constructor.name;
     const requestIdPrefix = formatRequestIdMessage(requestId);
     const currentDateNow = Date.now();
-    try {
-      const shouldLimit = this.hbarLimiter.shouldLimit(currentDateNow, SDKClient.transactionMode, callerName);
-      if (shouldLimit) {
-        throw predefined.HBAR_RATE_LIMIT_EXCEEDED;
-      }
-
-      this.logger.info(`${requestIdPrefix} Execute ${transactionType} transaction`);
-      const resp = await transaction.execute(this.clientMain);
-
-      this.logger.info(
-        `${requestIdPrefix} ${resp.transactionId} ${callerName} ${transactionType} status: ${Status.Success} (${Status.Success._code})`,
-      );
-      return resp;
-    } catch (e: any) {
```

Comment:
removing try catch block, because the same operation is performed in the `executeGetTransactionRecord()` method.

Reply:
Are we sure?
What happens when an executeTransaction fails for a PreCheck reason?
I believe that's what this check is for.

Reply:
Are we sure?
What happens when an executeTransaction fails for a PreCheck reason?
I believe that's what this check is for.

Reply:
It should throw and be caught one level above in eth.ts catch handler https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/src/lib/eth.ts#L1436